src/test/java/steppsDefinition/AmazonPageStepsDefinition.java

```diff
@@ -0,0 +1,50 @@
+package steppsDefinition;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class AmazonPageStepsDefinition {
+    WebDriver driver;
+
+    @Given("user is on Amazon home page")
+    public void user_is_on_home_page() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
```

Comment:
as a next step, that could go to a Hook method. When you will get to implement it later on.